"""creating users table to fix bugs

Revision ID: 92c0cdf83476
Revises: 234ec3766495
Create Date: 2023-01-09 11:17:35.454752

"""
from alembic import op
import sqlalchemy as sa


# revision identifiers, used by Alembic.
revision = '92c0cdf83476'
down_revision = '234ec3766495'
branch_labels = None
depends_on = None


def upgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.create_table('users',
    sa.Column('serial_no', sa.Integer(), nullable=False),
    sa.Column('id', sa.Integer(), nullable=False),
    sa.Column('first_name', sa.String(), nullable=False),
    sa.Column('last_name', sa.String(), nullable=False),
    sa.Column('email', sa.String(), nullable=False),
    sa.Column('country', sa.String(), nullable=False),
    sa.Column('phone_no', sa.Numeric(), nullable=False),
    sa.Column('role', sa.String(), nullable=False),
    sa.Column('capital', sa.String(), nullable=False),
    sa.Column('phase', sa.String(), nullable=False),
    sa.Column('mt_login', sa.String(), nullable=False),
    sa.Column('metatrader_password', sa.String(), nullable=False),
    sa.Column('mt_server', sa.String(), nullable=False),
    sa.Column('analytics', sa.String(), nullable=False),
    sa.Column('transaction_id', sa.String(), nullable=False),
    sa.Column('created_at', sa.TIMESTAMP(timezone=True), server_default=sa.text('now()'), nullable=False),
    sa.PrimaryKeyConstraint('serial_no'),
    sa.UniqueConstraint('email'),
    sa.UniqueConstraint('phone_no')
    )
    # ### end Alembic commands ###


def downgrade() -> None:
    # ### commands auto generated by Alembic - please adjust! ###
    op.drop_table('users')
    # ### end Alembic commands ###
